### PR TITLE
Web Views for Volunteer Home Page

### PR DESCRIPTION
**Pull request by mngshm**
The PR contains the Web Views for FOSS United Volunteer Home Page. 

## Web Template 1 
image

## Web Template 2 - Roles and benefits of Volunteers
- This section consists of Roles and benefits of a volunteer volunteering at FOSS United.
- Lemme know if we have to add more roles and benefits. 
- **Note**: The benefits section is a bit different from what was planned in the Prototype Design. I came up with the above implemented layout/design after experimenting. This looks way more cleaner than in the design I made. 

image

## Web Template 3 - Custom CTA
- The last section includes redirection links which will redirect ahead to Meetup Volunteer, Event Volunteer and NGO Volunteer form (which are to be added in the future). 
image

## Whole Page View and Responsive Mode screenshot
image
image

### CODE REVIEW

File: fossunited/volunteering/web_template/custom_cta/custom_cta.html
Comment (harshtandiya) [inline fossunited/volunteering/web_template/custom_cta/custom_cta.html:1]: Let's rename it to something more specific, Like "FOSS Volunteer - Custom CTA"

**Comment by harshtandiya** [comment]
Designs are very inconsistent with what we already have. How about we let it be re-designed and then merge to develop?

File: fossunited/volunteering/web_template/custom_cta/custom_cta.html
Comment (harshtandiya) [inline fossunited/volunteering/web_template/custom_cta/custom_cta.html:1]: The buttons are inconsistent in with, doesn't look good.

**Comment by mngshm** [comment]
Yes, lets make all the changes needed.

File: fossunited/volunteering/web_template/roles_and_benefits/roles_and_benefits.html
Comment (harshtandiya) [inline fossunited/volunteering/web_template/roles_and_benefits/roles_and_benefits.html:1]: The benefits section can use some more padding

File: fossunited/volunteering/web_template/volunteers_welcome_page/volunteers_welcome_page.html
Comment (harshtandiya) [inline fossunited/volunteering/web_template/volunteers_welcome_page/volunteers_welcome_page.html:1]: Component is redundant. Already have a template of the same kind

Comment (harshtandiya) [reply fossunited/volunteering/web_template/volunteers_welcome_page/volunteers_welcome_page.html:1]: Please, check the PRs when web templates are added. Also checkout templates in the staging

**Review by harshtandiya — CHANGES_REQUESTED** [review]

**Comment by Mangeshrex** [comment]
image

Please go through one more time and after that I'll start pushing the changes.

**Comment by mngshm** [comment]
image

@harshtandiya  Please go through this once, and then I'll start pushing the changes.

**Comment by harshtandiya** [comment]
@Mangeshrex 

- The text in the button is supposed to be inter font.

- I hope the hero image of the globe is not hardcoded. We already have it in landing page, and we should not repeat it here.

- We have a section similar to roles for volunteers. We can just use that. However, you can push this template section too. Just make sure you name it differently. 
image

- Same for the Benefits section. However, the design is definitely different, so we can have this as a different section. Name it likewise. The containers look a little claustrophobic, how about giving a little more padding-x?
image

- The buttons in the last section are again not of uniform width. Please change that. Maybe remove the border for the parent container? seems a little tacky?

**Comment by mngshm** [comment]
- Will fix the text font, I don't where does the issue actually lie, it was made using the template made by you. will go through it again 
- I don't understand what you mean by "hardcoding the globe", were you expecting me to use jinja templates and make fields for the roles ? 
- Yes, the roles and benefits section is named as "roles and benefits". And I've avoided making smaller and smaller templates. So, I've just made one template of the whole section.

**Comment by mngshm** [comment]
@harshtandiya have pushed the changes.

**Comment by mngshm** [comment]
image

**Comment by mngshm** [comment]
image

Final web page view with updated changes including: 
- fixing asset path: using fossunited/public for storing icon svg's 
- fixing the padding: fixed the padding above role icons and added it inside div for brevity 
- deleted FFFForward: because it was useless, as we are using Press2P tentatively.

**Review by harshtandiya — APPROVED** [review]

**Pull request merged**